server/pbench/bin/pbench-verify-backup-tarballs

```diff
@@ -91,8 +94,72 @@ if cd $backup ;then
     done < $controllers | sort > $listb
 fi
 
+# make sure the directories exist
+mkdir -p $backup/QUARANTINE
+mkdir -p $primary/QUARANTINE
+
+# function to calculate prefix name
+calculate_prefixname () {
+    tarname=${1##*/}
+    prefixname=${tarname%%.*}
+    despath=`echo ${1%/*} | sed 's;^\.;'$2';'`
+    prefixpath=`echo $prefixname | sed 's;^;'.prefix/prefix.';'`
+    prefix="$despath/$prefixpath"
+}
+
+# move the tarballs in backup, that fail the MD5 check to quarantine directory
```

Comment:
Does the comma (",") belong in this sentence?

Besides the grammar, perhaps we should consider revisiting the handling of the backup tar balls first.  In the case of a backup tar ball experiencing bit rot, if the archive tar ball is good, we don't need to quarantine the backup tar ball, we can just replace it with a fixed up backup.

So if we first validate all the archive tar balls, and for each one that is bad, we would then find its backup, verify the backup is good, and then replace the bad archive tar ball.  If the backup does not exist, or it is bad, then we would quarantine the archive tar ball, and remove the backup tar ball (if it exists).

Then for each archive tar ball that is good, we just need to verify that the backup tar ball is a duplicate.

Finally, we'll want a simple pass to ensure the list of backup tar balls matches the list of archive tar balls after all processing is complete.

Does that seem reasonable?